.github/CODEOWNERS

```diff
@@ -1 +0,0 @@
-* @mimir-maintainers
```

Comment:
I'm not sure what happened here

Reply:
I also see a lot of other changes. you can try starting clean from grafana:main onto a branch in your fork, cherry-pick aa62e3cad104e142ba019d71fa67cafa9d256225 3a37b08db679f11a1e66aee84285261dfa9ede21 and 20de41934fc7e752ead4bfb0a6a7129f7a435aaf and force push

Reply:
yeah, really messed up this one.